app/components/post_list/post/body/reactions/reaction.tsx

```diff
@@ -95,4 +97,4 @@ const Reaction = ({count, emojiName, highlight, onPress, onLongPress, theme}: Re
     );
 };
 
-export default Reaction;
```

Comment:
May not be needed in short term thanks to the memoization done on the Reactions component, but will probably be useful on other interactions, like adding or removing reactions, which will probably make use of this.